.env.example

```diff
@@ -0,0 +1 @@
+CURRENT_UID=1000
```

Comment:
Please add a blank line at the end of the file.

Reply:
Added.